tests/tests/tier0/bluechi-monitor-service-removed/main.fmf

```diff
@@ -0,0 +1,2 @@
+summary: Test if a removal of a service can be monitored using bluechictl monitor
```

Comment:
Its probably best to change the directory and test name to `bluechictl-monitor-service-removed` or something similar that includes the hint of using `bluechictl` since it could be (and is) done via the plain DBus API.

Reply:
Yeah, I was thinking about it, so let's rename this one and in different PR rename other existing tests to cleary identify bluechictl usage